app/src/forms/form/exportService.js

```diff
@@ -188,6 +188,7 @@ const service = {
     return SubmissionData.query()
       .column(service._submissionsColumns(form, params))
       .where('formId', form.id)
+      .where('deleted', false)
```

Comment:
This should not be hard coded IMHO - instead, this should come through the params argument so that it is configurable and can be overridden if required.

Reply:
This would be good for data exports yeah, Then the FE can just pass in an active or deleted flag

Reply:
done. just saw your comment Luke.. i called it showDeleted.. let me know if you want that changed